charts/alertmanager/values.yaml

```diff
@@ -37,14 +35,6 @@ dnsConfig: {}
   #   - name: ndots
   #     value: "2"
   #   - name: edns0
-securityContext:
```

Comment:
@ybettan instead of removing, we should introduce `enabled` flag which would by default be `false`

Reply:
Fixed.